ignite/distributed/comp_models/xla.py

```diff
@@ -50,17 +50,17 @@ def __init__(self, backend=None, **kwargs):
             else:
                 self._init_from_context()
 
-        def _create_from_backend(self, backend, **kwargs):
+        def _create_from_backend(self, backend: str, **_kwargs: Any) -> None:
```

Comment:
How about this rename, can we keep it as `kwargs` instead of `_kwargs` ?

Reply:
sure, I can change it back, I only renamed it, because the variable is not used and my IDE complains about it.